packages/vats/decentral-devnet-config.json

```diff
@@ -83,6 +125,18 @@
     "board": {
       "sourceSpec": "@agoric/vats/src/vat-board.js"
     },
+    "ibc": {
+      "sourceSpec": "@agoric/vats/src/vat-ibc.js"
+    },
+    "network": {
+      "sourceSpec": "@agoric/vats/src/vat-network.js"
+    },
+    "priceAuthority": {
+      "sourceSpec": "@agoric/vats/src/vat-priceAuthority.js"
+    },
```

Comment:
why does `vat-priceAuthority.js` belong here rather than in a vaults proposal?
(semi-rhetorical; I suppose I could figure it out if I looked around.)